### PR TITLE
Add basic schema and client functions for Network setting

### PR DESCRIPTION
**Pull request by Krishnan-Priyanshu**
# Description
To create client function for network setting resource consumptions.

Note: **_Resource CRUD functions are boilerplate code with empty definitions. We generated the code to document the network settings schema_**

# ISSUE TYPE
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### RESOURCE OR DATASOURCE NAME
network_setting_resource

##### OUTPUT

```
root@lglw9915:~/ome-network/terraform-provider-ome# /usr/local/go/bin/go test -coverprofile=coverage.out -timeout 90m -run TestNetwork terraform-provider-ome/clients
ok      terraform-provider-ome/clients  0.078s  coverage: 11.3% of statements
```
##### ADDITIONAL INFORMATION

```

```
# Checklist:
image

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

### CODE REVIEW

**Comment by terraform-providers-svc** [comment]
Can one of the admins verify this patch?

File: models/job.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package models
+
+// Job for getting job info.
+type Job struct {
```
Comment (rounak-adhikary) [inline models/job.go:4]: There is already a struct in clients/api_utility.go called JobResp. Looks like u have duplicated it.

File: clients/job.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+package clients
+
+import (
+	"fmt"
+	"terraform-provider-ome/models"
+)
+
+// GetJobByID to get job by ID.
+func (c *Client) GetJobByID(id int) (models.Job, error) {
```
Comment (rounak-adhikary) [inline clients/job.go:22]: This seems to be a duplication of clients/api_utility.go GetJob function. Please check.

File: models/network_setting.go [hunk 3]
```diff
@@ -0,0 +1,311 @@
+package models
+
+import "github.com/hashicorp/terraform-plugin-framework/types"
+
+// NetworkAdapterSetting for network adapter setting
+type NetworkAdapterSetting struct {
+	// OdataContext      string            `json:"@odata.context"`
+	// OdataType         string            `json:"@odata.type"`
```
Comment (rounak-adhikary) [inline models/network_setting.go:8]: Please remove these comments

File: clients/job.go (hunk 2 above)
Comment (Krishnan-Priyanshu) [reply clients/job.go]: removing job related files.

File: clients/network_setting.go [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+		return models.NetworkAdapterSetting{}, err
+	}
+
+	bodyData, err := c.GetBodyData(response.Body)
+	if err != nil {
+		return models.NetworkAdapterSetting{}, err
+	}
+	networkAdapterSetting := models.NetworkAdapterSetting{}
+	alpha := string(bodyData)
+	fmt.Println(alpha)
```
Comment (rounak-adhikary) [inline clients/network_setting.go:35]: pls remove this

Comment (rounak-adhikary) [reply clients/network_setting.go:35]: and other print sattements

**Review by rounak-adhikary — APPROVED** [review]
LGTM

**Review by AnikaAgiwal2711 — APPROVED** [review]

**Pull request merged**